docs/contributing/index.rst

```diff
@@ -63,11 +63,75 @@ And now you can use gulp for installing views, by just running a simple command:
 
     $ yarn run gulp
 
-For the contributing process questions, please refer to the `Contributing Guide <http://docs.sylius.org/en/latest/contributing/index.html>`_ that comes up in the following chapters.
+For the contributing process questions, please refer to the `Contributing Guide <http://docs.sylius.org/en/latest/contributing/index.html>`_ that comes up in the following chapters:
+
+.. toctree::
+    :maxdepth: 1
+
+    code/index
+    documentation/index
+    translations/index
 
 .. _Gulp: http://gulpjs.com/
 .. _Node.js: https://nodejs.org/en/download/
 
+
+How to contribute using Sylius Vagrant
```

Comment:
I think it would be much better to keep vagrant description in repo readme and here just create a link to the repo.

```
How to contribute using Sylius Vagrant

If you would like to contribute to the repo without worrying about environment, you can follow this guide to setup you vagrant machine
```